tests/test_install.ts

```diff
@@ -104,13 +107,13 @@ describe("Agama test", function () {
     afterEach(async function () {
         if (this.currentTest!.state === "failed") {
             // directory for storing the data
-            const dir = "log";
+            const dir: string = "log";
```

Comment:
this is not needed, it is inferred by TS, could you please remove this one and the other below, they don't give you any extra, directory name are also string literal and the return boolean can be read from the promise then part. Same for the asserts later, you get the inner text to get an string, type inferred is fine, we don't need to make verbose the TS.

Reply:
924e163 Solved